swapi_api/src/modules/characterDetails/characterDetails.scss

```diff
@@ -0,0 +1,21 @@
+@mixin characterContainer {
+    color: red;
+}
+
+@mixin characterInfo{
+    margin-bottom: 10px;
+}
+
+@mixin characterInfo_label{
```

Comment:
piszesz w scss, więc to jest zbędne. masz mixin characterInfo, więc w character info robisz 
```
&_label {
  font-weight: bold;
  margin-right: 5px;
 }
```